spec/resources/bank_accounts/bank_accounts_api_spec.rb

```diff
@@ -41,14 +41,14 @@
     end
 
     it "should apply given filters for find_by" do
-      expect(Elmas).to receive(:get).with("crm/BankAccounts?$filter=Account+eq+guid'12abcdef-1234-1234-1234-123456abcdef'&$filter=ID+eq+guid'12abcdef-1234-1234-1234-123456abcdef'")
+      expect(Elmas).to receive(:get).with("crm/BankAccounts?$filter=Account eq guid'12abcdef-1234-1234-1234-123456abcdef'&$filter=ID eq guid'12abcdef-1234-1234-1234-123456abcdef'")
```

Comment:
Are these URL's properly coded? I think it should be something like `%20` or am I missing something?

*This comment applies to all the URL changes*

Reply:
![Screenshot 2022-09-01 at 10 04 18](https://user-images.githubusercontent.com/5468309/187864395-93d53e1c-ec55-4d9f-bed7-f1da593d6178.png)

Reply:
Oh .. I did not say anything 😂